File: library/src/main/java/cn/cqray/android/dialog/amin/FadeOut.java
package cn.cqray.android.dialog.amin;

import androidx.annotation.NonNull;

import cn.cqray.android.anim.AnimatorBuilder;

/**
 * 淡出
 * @author Cqray
 */
public class FadeOut extends DialogAnimator {

    @Override
    public void onHandle(@NonNull AnimatorBuilder builder) {
        builder.fadeOut();
    }
}

File: library/src/main/java/cn/cqray/android/dialog/amin/FadeOut.kt
package cn.cqray.android.dialog.amin

import cn.cqray.android.anim.ViewAnimator

/**
 * 淡出
 * @author Cqray
 */
class FadeOut : GetAnimator() {
    override fun onAnimatorPrepared(animator: ViewAnimator) {
        animator.fadeOut()
    }
}